docker-compose.yml

```diff
@@ -9,15 +9,55 @@ services:
             - SPLUNK_HEC_TOKEN=11111111-1111-1111-1111-1111111111113
             - SPLUNK_PASSWORD=changed!
         ports:
-            - 8000:8000
-            - 8088:8088
-            - 8089:8089
+            - "8000:8000"
+            - "8088:8088"
+            - "8089:8089"
         volumes:
+            # Weather App
             - ./setup_pages/weather_app_example:/opt/splunk/etc/apps/weather_app_example
+            - ./lib/splunklib:/opt/splunk/etc/apps/weather_app_example/lib/splunklib
```

Comment:
Thanks for updating the weather app as well - can you remove the `make modules` Makefile command since it's no longer needed and make the updates to the `weather_app_example` to import from /lib rather than `vendor`?

Reply:
@fantavlik `weather_app_example` also use `requests` library, so if we want to generalise the dependency installation we need to use same requirements.txt in `make up` command.
Let me know if you want to have this, I'll make the changes.

Reply:
ah good catch - I think that makes sense to move everything in the single `make up` command yes let's go ahead with that

Reply:
Done